server/index.js

```diff
@@ -37,6 +38,23 @@ app.get('/api/products', (req, res, next) => {
     .catch(err => next(err));
 });
 
+app.get('/api/products/:productId', (req, res, next) => {
+  const productId = parseInt(req.params.productId, 10);
```

Comment:
You should validate that the `productId` is a positive integer as we've done in prior exercises.

Reply:
Sounds good.